docs/source/concepts/calibration/EngineeringCalibration.rst

```diff
@@ -0,0 +1,74 @@
+.. _Engineering Calibration:
+
+Engineering Calibration
+=======================
+  
+.. contents::
+  :local:
+
+
+Calculating Calibration
+-----------------------
+
+Notes
+#####
+
+* The engineering Calibration is currently implemented to support Engin-X at ISIS, it may work for other instruments, but is subject to change at this point.
+
+Data Required
+#############
+
+You will need to run a sample which will create good clean peaks at known reference dSpacing positions.  To get a good calibration you will want good statistics on this calibration data.  Vanadium and Ceria are common samples used for calibration.
+
+.. interface:: Engineering Diffraction
+  :align: right
+  :width: 400
+
+Using the Engineering GUI
+#########################
+
+Mantid rpovides a graphical user interface that  calculate can calibrations and
+visualize them.
+
+It is possible to load an existing calibration (as a CSV file) and to
+generate a new calibration file (which becomes the new current
+calibration).
+
+A description of the inte3rface and all of it's controls can be found 
+:ref:`here <ui engineering calibration>`.
+
+Calibrating the entire instrument
+#################################
+
+Calibration of every detector/pixel at the same time is done by :ref:`EnggCalibrateFull <algm-EnggCalibrateFull>`.  This calculates the calibration in a single step and outputs a table workspace of the calibration.  As it is calibrating the whole intrstrument it can take some time, the next section shows howw you can perform a partial calibration.
+
+Calibrating individual banks
+############################
+
+:ref:`EnggCalibrate <algm-EnggCalibrate>` allows calibration of indiviual bank within the instrument, which is usefull if only one bank has moved from a previous calibration, or if the calibration failed for a certain area of the instrument.  
+  
+Under the hood
+##############
+
+All of these approaches use the algorithm :ref:`EnggFitPeaks <algm-EnggFitPeaks>`, :ref:`FindPeaks <algm-FindPeaks>` and  :ref:`EnggVanadiumCorrections <algm-EnggVanadiumCorrections>` to find, and fit the recorded peaks and compare them to the expected values.  
```

Comment:
There's been a recent change in the algorithms and the old `EnggFitPeaks` has been split into `EnggFitPeaks` and `EnggFitDIFCFromPeaks`. So where `EnggFitPeaks` is mentioned `EnggFitDIFCFromPeaks` should also be listed.
This is not important and can be edited later on.

When this is in master I'd update the documentation of the Engg GUI to link to this document about calibration.